ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -24,7 +24,27 @@ resources:
       memory: 5Gi
 
 tests:
-- as: yaml-lint
-  commands: dist/prow_yaml_lint.sh
-  container:
-    from: src
+  - as: yaml-lint
+    commands: dist/prow_yaml_lint.sh
+    container:
+      from: src
+  - as: rustfmt
```

Comment:
I think we should add the explicit version here too.

Reply:
Good idea, this would make future diffs look cleaner. Done